openapi.yaml

```diff
@@ -0,0 +1,698 @@
+swagger: '2.0'
+info:
+  title: 'Game Curator API'
+  description: 'Learning project API to retrieve and manipulate game review data.'
+  version: 'v1'
```

Comment:
These strings are no need to be quoted. When writing YAML, we only quote when needed. Ref: https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#quoting-conventions

Can you go over your openapi file and fix them all to make sure your code is consistent?

Reply:
Fixed inconsistent single quote usage.